source/models/the-rest-adapter.md

```diff
@@ -5,6 +5,15 @@ associated with each model are conventional; this means that, if you
 follow the rules, you will not need to configure the adapter or write
 any code in order to get started.
 
+#### Customizing the adapter
```

Comment:
Let's take away the heading here - it unnecessarily breaks up the page flow, and it doesn't start something that could be considered a new section.